libraries/default.rb

```diff
@@ -428,17 +491,22 @@ def initialize_database(current_deploy_root)
       end
     end
 
-    def install_init_scripts(current_deploy_root, matterhorn_repo_root)
-      log_dir = node.fetch(:matterhorn_log_directory, '/var/log/matterhorn')
-
+    def xmx_ram_for_this_node(xmx_ram_ratio)
```

Comment:
Agreed. The diff is really hard to read, but I think the end result makes sense.